app/Actions/Photo/Pipes/Standalone/SetChecksum.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Actions\Photo\Pipes\Standalone;
+
+use App\Contracts\PhotoCreate\StandalonePipe;
+use App\DTO\PhotoCreate\StandaloneDTO;
+
+class SetChecksum implements StandalonePipe
+{
+	public function handle(StandaloneDTO $state, \Closure $next): StandaloneDTO
+	{
+		// The original and final checksum may differ, if the photo has
+		// been rotated by `putSourceIntoFinalDestination` while being
+		// moved into final position.
+		$state->photo->checksum = $state->streamStat->checksum;
```

Comment:
This is already done in PlacePhoto line 30? 
https://github.com/LycheeOrg/Lychee/pull/2371/files#diff-39105fae9f03961634b4eab092e28bea16a8b0864bda8f60c9d2a5a93fa66d69R30

Reply:
I will double check.

Reply:
Fixed by removing the code in `PlacePhoto`.